<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,8">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>AcadEref</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>物流
项目</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/res/delivery.png</normaloff>:/res/delivery.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton2">
         <property name="font">
          <font>
           <family>AcadEref</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>车库
项目</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/res/Garage.png</normaloff>:/res/Garage.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton3">
         <property name="font">
          <font>
           <family>AcadEref</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>研发
项目</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/res/development.png</normaloff>:/res/development.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton4">
         <property name="text">
          <string>PushButton</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Project"/>
   <addaction name="separator"/>
   <addaction name="action_AGV"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Project">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/project manager .png</normaloff>:/res/project manager .png</iconset>
   </property>
   <property name="text">
    <string>项目管理</string>
   </property>
   <property name="toolTip">
    <string>项目管理</string>
   </property>
  </action>
  <action name="action_AGV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/agv.png</normaloff>:/res/agv.png</iconset>
   </property>
   <property name="text">
    <string>自主AGV</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
